timeline.csv

```diff
@@ -2,6 +2,7 @@ start,end,title,description,eventType,eventImportance,referenceURI,imageURI,medi
 1950,1970,Software shared as public domain,"While not called ""open source"" at the time, researchers would collaborate on code and considered it communal knowledge",other,note,https://en.wikipedia.org/wiki/Public-domain_software#Early_academic_public-domain_software_ecosystem,https://upload.wikimedia.org/wikipedia/commons/thumb/3/39/Cc-public_domain_mark_white.svg/768px-Cc-public_domain_mark_white.svg.png,,"era,academia,culture,public domain"
 1969,,ARPANET is created,"Created in 1969 and shut down in 1990, the Advanced Research Projects Agency Network (ARPANET) was a network project that the United States agency ARPA (later known as DARPA) funded in 1966 with the goal of enabling communication and access to remote systems. An additional goal of ARPANET was to facilitate the distribution of software. ARPANET initially implemented the 1822 protocol, then switched to the Network Control Protocol (NCP), before transitioning to TCP/IP. ARPANET is generally considered to be one of the predecessors to the Internet.",release,critical,https://web.archive.org/web/20120915113839/http://www.darpa.mil/WorkArea/DownloadAsset.aspx?id=2554,https://commons.wikimedia.org/wiki/File:Arpanet_in_the_1970s.png#/media/File:Arpanet_in_the_1970s.png,,"ARPA,internet,network,government,tcp/ip,1822 protocol"
 1974,1977,Commission on New Technological Uses of Copyrighted Works recommended that software should be protected by copyright law,"The Commission on New Technological Uses of Copyrighted Works (or CONTU) was a three-year study on copyright in regards to computers and ""machine duplication systems"", with the goal of revising copyright laws in the United States. CONTU recommended that software be subject to copyright, with one dissention expressing that an executable program itself should not be copyrightable as it counts as a ""machine-control element"". The Computer Software Copyright Act of 1980 implemented much of CONTU's recommendations.",milestone,critical,https://repository.law.uic.edu/cgi/viewcontent.cgi?article=1573&context=jitpl,https://en.wikipedia.org/wiki/File:CONTU_portrait_1978.jpg,,"copyright,United States,policy"
+1980,,"The U.S. Department of Veteran Affairs begins development of the VistA electronic health record software under a public domain license","After performing research into electronic health records during the 1970s, and using a development methodology that involves pairing of programmers with clinicians during implementation, the Department of Veteran Affairs licenses their VistA healthcare information system software into the public domain",milestone,medium,https://www.oit.va.gov/about/history.cfm,,,"era,sharing,public domain,government,healthcare"
```

Comment:
I might disagree with the tag of "era" given that it doesn't have an end date. Otherwise, I learned something today -- thank you!